assignment_1/Concentration.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  Concentration.swift
+//  assignment_1
+//
+//  Created by Chen Shoresh on 15/06/2021.
+//
+
+import Foundation
+
+class Concentration{
+    
+    var cards = [Card]()
+    
+    var indexOfOneCardFaceUp : Int?
+    
+    var flipCount = 0
+    
+    var score = 0
+    
+    var seenCards = [Int]()
+    
+    
+    func chooseCard ( at index: Int)-> (Int,Int){
```

Comment:
Please split the logic to methods - for example, a method for `checkIfCardMatch()`

Reply:
```suggestion
    func chooseCard ( at index: Int) {
```